pages/common/jps.md

```diff
@@ -0,0 +1,23 @@
+# jps
+
+> Show JVM Process Status of current user.
+
+- List all JVM processes:
+
+`jps`
+
+- List all JVM processes with only PID:
```

Comment:
What is meant by `only PID`?

Reply:
It means `jps -q` will only show PID (Process IDentifier) of JVM processes, without any other Information like package name, arguments etc.